src/index.ts

```diff
@@ -202,31 +217,42 @@ export async function importActorProfile(
         try {
           if (fileName.endsWith('.json')) {
             result[fileName] = JSON.parse(content)
+            console.log('Parsed JSON file successfully:', fileName)
           } else if (fileName.endsWith('.yaml') || fileName.endsWith('.yml')) {
             result[fileName] = YAML.parse(content)
           } else if (fileName.endsWith('.csv')) {
             result[fileName] = content
+          } else {
+            console.warn(`Unsupported file type: ${fileName}, skipping...`)
           }
         } catch (error: any) {
-          reject(new Error(`Error processing file ${fileName}: ${error}`))
+          console.error(`Error processing file ${fileName}:`, error.message)
+        } finally {
+          next() // Always continue
         }
-        next()
       })
 
       stream.on('error', (error: any) => {
-        reject(new Error(`Stream error on file ${fileName}: ${error}`))
+        console.error(`Stream error on file ${fileName}:`, error.message)
+        next() // Continue even on stream error
       })
     })
 
     extract.on('finish', () => {
+      console.log('All files processed successfully.')
       resolve(result)
     })
 
     extract.on('error', (error) => {
-      reject(new Error(`Error during extraction: ${error}`))
+      console.error('Error during tar extraction:', error.message)
+      reject(new Error('Failed to extract tar file.'))
+    })
+
+    tarStream.on('error', (error) => {
+      console.error('Error in tar stream:', error.message)
```

Comment:
A lot of these logs could probably be moved into `cli.ts` and listen for `error` events on the stream in there.

imho we should only console.log in the cli invocation of this function, not every time this function is called